### DISCUSSION TITLE
Script Closes without Output (Windows 11)

### DISCUSSION THREAD

**Discussion by MarcusKirchdoerfer** [illsk1lls/ZipRipper#52] [Q&A]
Hi! First of all thank you for the script! 

I have an issue running the script though. When I double click the file, a CMD promt open and closes instantly. If I open CMD (or PowerShell), navigate to the script and run it, same result, new line without any output. In windows task manager CMD or PowerShell shows up normaly wihtout any CPU or GPU time. Same when I start it with Admin rights. So I am assuming at this point that something fails and there is no error message. I have tried an exception with my antivirus but no dice. On the firewall I also do not see any activity. 

System
Windows 11 professionall  (latest patch installed)
AMD AMD Ryzen 5 5600
NVIDIA RTX 3080
Bitdefender as a Virus scanner

Any idea what else I can try? It drives me nuts that there is no error or anything.

Thx for the help!

Forgot to add that I have applied the AMD "fix" under .ressources

**Comment by illsk1lls** [comment]
im not sure, have you tried running it with AV completely disabled? the code is fully unobfuscated, i know its a bit difficult to read for beginners but it is safe to use... 

it may be flagging the self elevating or powershell behaviors to be safe

**Reply by MarcusKirchdoerfer** [reply]
Ill give it a try tonight.

**Reply by MarcusKirchdoerfer** [reply]
Disabling all features in Bitfdefender did the trick. It is now doing it´s thing. Thx for the help and the script!

**Reply by illsk1lls** [reply]
Glad it worked out, good luck with the file
